content/W2017/W17-new-committee-applications-S17.md

```diff
@@ -0,0 +1,28 @@
+Title: Spring 2017 WiCS Committee Applications
```

Comment:
The file name is a lil funky. To change just do "mv <oldfilename> <newfilename>"